src/pages/ClaimReward.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * This program has been developed by students from the bachelor Computer Science at Utrecht University within the Software Project course.
+ * © Copyright Utrecht University (Department of Information and Computing Sciences)
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export type ClaimRewardData = {
+  division: number;
+};
+
+import { Controller, useForm, useWatch } from 'react-hook-form';
+import { Label } from '../components/ui/Label';
+import { Slider } from '../components/ui/Slider';
+import { Button } from '../components/ui/Button';
+import TokenAmount from '../components/ui/TokenAmount';
+import { MainCard } from '../components/ui/MainCard';
+import { IconBaseProps } from 'react-icons/lib';
+import { HiOutlineCurrencyDollar } from 'react-icons/hi2';
+import { Card } from '../components/ui/Card';
+import { TOKENS } from '../lib/constants/tokens';
+
+/*
+ * repToBeClaimed is the total amount of reputation the user may claim
```

Comment:
Use the structure we have used before for comments like this, so:
```js
 * @param props.repToBeClaimed Total amount of reputation the user can claim
```

Also, I would personally rename `repToBeClaimed` to `claimableRep`